### PR TITLE
Update radio connections

### PR DESCRIPTION
**Pull request by bbrk24**
This PR updates the diagrams and code to reflect the actual radio board we will be using.

### CODE REVIEW

**Comment by bbrk24** [comment]
To be clear, the resistors on the data lines only works because it's UART. This wouldn't work for I2C.

File: sketch_oct9a.ino [hunk 1]
```diff
@@ -119,17 +119,6 @@ void updateSDCardLEDs() {
 }
 
 void setup() {
```
Comment (DarthSethious1) [inline sketch_oct9a.ino:121]: why is all of this code gone now?

Comment (bbrk24) [reply sketch_oct9a.ino:121]: It was setup for the RTS and CTS pins. Now that we're only using RX/TX and not RTS/CTS, this isn't necessary.

Comment (DarthSethious1) [reply sketch_oct9a.ino:121]: IDEK what RTS and CTS are, so thats fine by me

Comment (bbrk24) [reply sketch_oct9a.ino:121]: Extra control flow stuff for USART, they're completely optional.

**Pull request merged**